components/MarkdownEditor/InternalEditor.vue

```diff
@@ -90,7 +90,7 @@
     </div>
     <div class="p-1">
       <Milkdown
-        :class="markdownClasses"
+        :class="markdownSmClasses"
```

Comment:
Could we add explicit imports for this? If we migrate this component to datagouv/components it will stop working in `udata-front-kit`